src/main/java/duke/commands/ListCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.commands;
+
+import java.util.ArrayList;
+
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Lists all tasks in the task list.
+ */
+public class ListCommand extends Command {
+    /**
+     * Executes the list command.
+     */
+    @Override
+    public String execute(TaskList taskList, Storage storage) {
+        String response = "";
+        int length = taskList.size();
+        StringBuilder sb = new StringBuilder();
+
+        if (length == 0) {
```

Comment:
this part is repeated for all subclasses of Command, maybe you could move it to Command class?